src/screens/Home/Home.js

```diff
@@ -447,6 +456,16 @@ class HomeScreen extends React.Component<Props, State> {
               headerComponent={(
                 <React.Fragment>
                   <WalletsPart handleWalletChange={this.handleWalletChange} />
+                  {!!walletConnectRequests &&
+                  <RequestsWrapper marginOnTop={walletConnectRequests.length === 1}>
+                    {walletConnectRequests.length > 1 &&
+                    <ButtonText
```

Comment:
For some reason on Android it's centered o.o

Reply:
Fixed, thanks! (Checked both on Android and on iOS)